Obsidian/WorldData/World.cs

```diff
@@ -483,5 +483,22 @@ internal bool TryAddEntity(Entity entity)
 
             return region.Entities.TryAdd(entity.EntityId, entity);
         }
+
+        public async Task SpawnExperienceOrb(PositionF position, short count)
+        {
+            await this.Server.BroadcastPacketAsync(new SpawnExperienceOrb
+            {
+                Count = count,
+                Position = position,
+                EntityId = 2 
+                // 2 = Experience Orb Source: https://minecraft.gamepedia.com/Java_Edition_data_values/Pre-flattening/Entity_IDs
+            });
+        }
+
+        public async Task SpawnPainting(Position position, Painting painting, PaintingDirection direction, Guid uuid = default)
+        {
+            if (uuid == Guid.Empty) uuid = Guid.NewGuid();
+            await this.Server.BroadcastPacketAsync(new SpawnPainting(9, uuid, painting.Id, position, direction));
```

Comment:
I wonder where does the magic `9` come from?

Reply:
> I wonder where does the magic `9` come from?

The comment above seems related, https://minecraft.gamepedia.com/Java_Edition_data_values/Pre-flattening/Entity_IDs
says that 9 is the Entity ID for paintings.

Reply:
If the entity ID for paintings is _always_ 9, it should be in `SpawnPainting.cs` and setting a different entity ID shouldn't even be an option in the constructor. I'd say either hard code it in the `SpawnPainting` constructor, on one of it's properties or look if there's any loaded data for entity IDs in the registry.

Reply:
Addition to that, if the entity id for paintings is static then it's kinda weird that mojang defines it in the packets. hmm